src/components/GlobalPreferences/Notifications/notification-service-api.js

```diff
@@ -11,7 +11,7 @@ import {
 
 // The notifications API expects mainnet or rinkeby. This deviates from web3's getNetworkType which returns main
```

Comment:
can we replace `web3 getNetworkType` with `use-wallet`'s networkType ?

Reply:
sure, I'll create a reminder ticket to fix the backend service to accept `main` instead of `mainnet`

Reply:
nope, nope... I meant the comment, not the actual code...